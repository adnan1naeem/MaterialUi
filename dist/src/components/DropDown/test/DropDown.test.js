"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
const __1 = require("../");
const react_1 = __importDefault(require("react"));
const enzyme_adapter_react_16_1 = __importDefault(require("enzyme-adapter-react-16"));
const enzyme_1 = require("enzyme");
enzyme_1.configure({ adapter: new enzyme_adapter_react_16_1.default() });
test('it should show normal dropDown', () => {
    const DropdownList = enzyme_1.shallow(react_1.default.createElement(__1.Dropdown, { value: 'jsbha' }));
    expect(DropdownList.props().value).toEqual('jsbha');
});
test('it should show dropdown with autoWidth true', () => {
    const DropdownList = enzyme_1.shallow(react_1.default.createElement(__1.Dropdown, { autoWidth: true }));
    expect(DropdownList.props().autoWidth).toEqual(true);
});
test('it should  disable dropdown with true prop', () => {
    const DropdownList = enzyme_1.shallow(react_1.default.createElement(__1.Dropdown, { disabled: true }));
    expect(DropdownList.props().disabled).toEqual(true);
});
test('it should  disable dropdown with true prop', () => {
    const DropdownList = enzyme_1.shallow(react_1.default.createElement(__1.Dropdown, { disabled: true }));
    expect(DropdownList.props().disabled).toEqual(true);
});
test('it should add multiple items with this props', () => {
    const DropdownList = enzyme_1.shallow(react_1.default.createElement(__1.Dropdown, { multiple: true }));
    expect(DropdownList.props().multiple).toEqual(true);
});
//# sourceMappingURL=DropDown.test.js.map