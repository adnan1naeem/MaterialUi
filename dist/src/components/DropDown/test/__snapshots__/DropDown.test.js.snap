// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should show normal dropDown 1`] = `
<div
  className="MuiInput-root-6 MuiInput-inkbar-8 MuiInput-underline-12"
  onBlur={[Function]}
  onFocus={[Function]}
  onSelect={undefined}
>
  <div
    className="MuiSelect-root-1"
  >
    <div
      aria-haspopup="true"
      aria-owns={null}
      aria-pressed="false"
      className="MuiSelect-select-2 MuiSelect-selectMenu-3 MuiInput-input-15 MuiInput-inputSingleline-18"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={undefined}
      onKeyDown={[Function]}
      role="button"
      tabIndex={0}
    >
      
    </div>
    <input
      aria-invalid={undefined}
      aria-required={undefined}
      autoComplete={undefined}
      autoFocus={undefined}
      defaultValue={undefined}
      id={undefined}
      name={undefined}
      onKeyDown={undefined}
      onKeyUp={undefined}
      placeholder={undefined}
      readOnly={undefined}
      required={undefined}
      rows={undefined}
      type="hidden"
      value="jsbha"
    />
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-21 MuiSelect-icon-5"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M7 10l5 5 5-5z"
      />
    </svg>
  </div>
</div>
`;

exports[`with color prop 1`] = `
<div
  className="MuiInput-root-6 MuiInput-inkbar-8 MuiInput-underline-12"
  onBlur={[Function]}
  onFocus={[Function]}
  onSelect={undefined}
>
  <div
    className="MuiSelect-root-1"
  >
    <div
      aria-haspopup="true"
      aria-owns={null}
      aria-pressed="false"
      className="MuiSelect-select-2 MuiSelect-selectMenu-3 MuiInput-input-15 MuiInput-inputSingleline-18"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={undefined}
      onKeyDown={[Function]}
      role="button"
      tabIndex={0}
    >
      
    </div>
    <input
      aria-invalid={undefined}
      aria-required={undefined}
      autoComplete={undefined}
      autoFocus={undefined}
      defaultValue={undefined}
      id={undefined}
      name={undefined}
      onKeyDown={undefined}
      onKeyUp={undefined}
      placeholder={undefined}
      readOnly={undefined}
      required={undefined}
      rows={undefined}
      type="hidden"
      value="jsbha"
    />
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-21 MuiSelect-icon-5"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M7 10l5 5 5-5z"
      />
    </svg>
  </div>
</div>
`;

exports[`with onSelelct prop 1`] = `
<div
  className="MuiInput-root-6 MuiInput-inkbar-8 MuiInput-underline-12"
  onBlur={[Function]}
  onFocus={[Function]}
  onSelect={undefined}
>
  <div
    className="MuiSelect-root-1"
  >
    <div
      aria-haspopup="true"
      aria-owns={null}
      aria-pressed="false"
      className="MuiSelect-select-2 MuiSelect-selectMenu-3 MuiInput-input-15 MuiInput-inputSingleline-18"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={undefined}
      onKeyDown={[Function]}
      role="button"
      tabIndex={0}
    >
      
    </div>
    <input
      aria-invalid={undefined}
      aria-required={undefined}
      autoComplete={undefined}
      autoFocus={undefined}
      defaultValue={undefined}
      id={undefined}
      name={undefined}
      onKeyDown={undefined}
      onKeyUp={undefined}
      placeholder={undefined}
      readOnly={undefined}
      required={undefined}
      rows={undefined}
      type="hidden"
      value="jsbha"
    />
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-21 MuiSelect-icon-5"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M7 10l5 5 5-5z"
      />
    </svg>
  </div>
</div>
`;
